dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5901,17 +5395,6 @@ public String toString() {
               + "before this file is persisted.")
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_FILE_SEQUENTIAL_PREAD_THRESHOLD =
-      dataSizeBuilder(Name.USER_FILE_SEQUENTIAL_PREAD_THRESHOLD)
```

Comment:
I didn't know this even existed. Looks like the optimization is something that should be preserved in Dora. But it's ok to remove this property key